client/components/suggestion/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import omit from 'lodash/omit';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import Button from 'components/button';
+import Summary from 'components/summary';
+import { translate as __ } from 'lib/mixins/i18n';
+
+const RadioButton = ( props ) => {
+	return (
+		<FormLabel>
+			<FormRadio { ...omit( props, 'children' ) } />
+			{ props.children }
+		</FormLabel>
+	);
+};
+
+const Suggestion = ( { acceptSuggestion, formValues, formActions, layout, suggestions } ) => {
+	return (
+		<div>
+			<RadioButton
+				checked={ ! acceptSuggestion }
+				onChange={ () => formActions.setFormProperty( 'acceptSuggestion', false ) } >
```

Comment:
This (suggestion) is a low level component. Should the onChange callback be passed in as a prop instead of defined here to avoid tight coupling to our form implementation?